lib/run-context.js

```diff
@@ -119,7 +119,9 @@ RunContext.prototype._run = function () {
  */
 RunContext.prototype.toPromise = function () {
   return new Promise(function (resolve, reject) {
-    this.on('end', resolve);
+    this.on('end', function () {
+      resolve(this.targetDirectory);
```

Comment:
Updated. I was getting that error on plain 1.3.0. The updated code fails on all 4 versions. I'm not sure what's wrong there, but I'll look into it.